plugin/src/main/kotlin/nl/avisi/anowalke/dto/ExpressionsDto.kt

```diff
@@ -0,0 +1,5 @@
+package nl.avisi.anowalke.dto
+
+data class ExpressionsDto(
+    val expressions: List<String>
```

Comment:
Ik weet niet of het in dit geval praktisch gaat zijn, je krijgt nu wel gedeelde logica tussen je api en plugin die je nu dubbel gaat definiëren. Je kan een aparte module maken die dit soort models bevat. Zo hoef je ze maar een keer te definiëren en te onderhouden.

Reply:
Ja ik had er wel over nagedacht om een shared module te maken maak dat leek me ook best wel weer extra werk dus toen heb ik het maar even zo gelaten. Of iig extra werk xD Veel zal het niet heel veel zijn.. Er zou in dat geval wss wel echt alleen die ene dto in zitten..

Reply:
Denk dat het relatief weinig werk is aangezien je al een opzet hebt voor meerdere modules. En daarbij snel veel toegevoegde waarde naar de toekomst toe. Ook weer een design decision van te maken